### PR TITLE
doc/cephadm: improve install guide

### PR DESCRIPTION
**Pull request by zdover23**
- Repair broken lists.
- Add bash prompt directives.
- Add code-block directives.
- Add command markup.
- Improve English grammar.

Fixes: https://tracker.ceph.com/issues/48023
Signed-off-by: Zac Dover <zac.dover@gmail.com>

## Checklist
- [x] References tracker ticket
- [x] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

**Comment by zdover23** [comment]
jenkins render docs

**Review by jdurgin — APPROVED** [review]

File: doc/cephadm/install.rst [hunk 1]
```diff
@@ -382,28 +481,40 @@ that configuration isn't already in place (usually in the
...
+  .. prompt:: bash #
+
+    ceph orch apply rgw *<realm-name>* *<zone-name>* --placement="*<num-daemons>* [*<host1>* ...]"
 
-  # ceph orch apply rgw *<realm-name>* *<zone-name>* --placement="*<num-daemons>* [*<host1>* ...]"
+For example, to deploy 2 rgw daemons serving the *myorg* realm and the *us-east-1* zone on *myhost1* and *myhost2*:
 
-For example, to deploy 2 rgw daemons serving the *myorg* realm and the *us-east-1*
-zone on *myhost1* and *myhost2*::
+  .. prompt:: bash #
```
Comment (tchaikov) [inline doc/cephadm/install.rst:492]: is the indent intended? see https://ceph--37867.org.readthedocs.build/en/37867/cephadm/install/#deploy-rgws .

Comment (zdover23) [reply doc/cephadm/install.rst:492]: It wasn't. Good catch. This comment inspired me to read through the document again, correcting other similar unintended indentations.

I'll resolve this issue, push and squash, and request your review again, Kefu.

**Comment by zdover23** [comment]
jenkins render docs

**Review by neha-ojha — APPROVED** [review]

**Pull request merged**